webpack.config.js

```diff
@@ -97,10 +105,15 @@ module.exports = {
             {from: 'intl', to: 'js'}
         ]),
         new webpack.optimize.UglifyJsPlugin({
+            mangle: true,
```

Comment:
oh interesting! Keeping it is fine with me either way – even though it's true by default I like that it's explicit, in case it leads to any issues in the future.